modules/ROOT/pages/use-tags-in-visualizer.adoc

```diff
@@ -44,7 +44,7 @@ You cannot apply the same tag multiple times to a node. All tags are converted t
 +
 image::create-tag.png[]
 +
-_Result:_ The tags that you added to a node is listed in the *Customization* panel. This node is included in the list of services if you xref:view.adoc#customize-view[filter by this tag].
+_Result:_ The tags that you added to a node is listed in the *Customization* section. This node is included in the list of services if you xref:view.adoc#customize-view[filter by this tag].
```

Comment:
the tags _are_ listed (leftover grammatical error)

Reply:
How embarrassing!! Accepted.